common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -821,15 +821,22 @@ StScrollBar {
     background-color: transparent;
     border: none;
 
-    .panel-button {
-      color: $fg_color;
-      &:hover { color: $fg_color; }
+    .panel-button,
+    .panel-button.clock-display .clock {
       &:focus, &:active, &:checked {
-        color: $selected_fg_color;
```

Comment:
Don't remove this. Otherwise the `$_shell_fg_color `will be used from the rule below, and everything with `$selected_bg_color` as a background must have `$selected_fg_color` foreground (even if both colors are currently the same).

Reply:
Missed that, corrected